components/endpoint-configuration-management/org.wso2.carbon.identity.api.server.endpoint.configuration.management/org.wso2.carbon.identity.api.server.endpoint.configuration.management.common/src/main/java/org/wso2/carbon/identity/api/server/endpoint/configuration/management/common/EndpointConfigurationManagementConstants.java

```diff
@@ -0,0 +1,106 @@
+    public static final String PLUS = "+";
+    public static final String URL_ENCODED_SPACE = "%20";
+    public static final String RESOURCE_NOT_EXISTS_ERROR_CODE = "CONFIGM_00017";
+    public static final String CONFIG_MGT_ERROR_CODE_DELIMITER = "_";
+    public static final String CORRELATION_ID_MDC = "Correlation-ID";
+    public static final String SERVER_API_PATH_COMPONENT = "/api/server";
+    public static final String V1_API_PATH_COMPONENT = "/v1";
+
+    public static final String TENANT_NAME_FROM_CONTEXT = "TenantNameFromContext";
+    public static final String TENANT_CONTEXT_PATH_COMPONENT = "/t/%s";
+
+    // endpoint configuration main properties.
+    public static final String ENDPOINT_URL = "endPointURL";
+    public static final String AUTH_TYPE = "authType";
+
+    /**
+     * Enums for error messages.
+     */
+    public enum ErrorMessage {
+
+        ERROR_CODE_END_POINT_URI_NOT_SPECIFIED("CMT-60001", "Empty end point URI", "End point URI is " +
+                "not specified in the request"),
+        ERROR_CODE_REFERENCE_NAME_NOT_SPECIFIED("CMT-60002", "Empty reference name",
+                "End point reference name is not specified in the request"),
+        ERROR_CODE_AUTH_TYPE_URI_NOT_SPECIFIED("CMT-60003", "Empty auth type", "Auth type is " +
+                "not specified in the request"),
+        // Client errors 600xx.
+        ERROR_CODE_CONFLICT_CONFIGURATION("60002", "Configuration already exists.",
+                "There exists a endpoint Configuration: %s in the tenant."),
+
+        // Server errors 650xx.
+        ERROR_CODE_ERROR_GETTING_ENDPOINT_CONFIGURATION("65003", "Error while getting endpoint configurations.",
+                "Error while retrieving endpoint configurations resource: %s."),
+        ERROR_CODE_ERROR_ADDING_ENDPOINT_CONFIGURATION("65004", "Unable to add endpoint configuration.",
+                "Server encountered an error while adding the endpoint configuration: %s"),
+        ERROR_CODE_ERROR_DELETING_ENDPOINT_CONFIGURATION("65005", "Unable to delete endpoint configuration.",
+                "Server encountered an error while deleting the endpoint configuration: %s"),
+        ERROR_CODE_ERROR_GETTING_ENDPOINT_CONFIGURATION_BY_TYPE("65006", "Error while getting endpoint configuration.",
+                "Error while retrieving %s endpoint configuration."),
+        ERROR_CODE_ERROR_UPDATING_ENDPOINT_CONFIGURATION("65007", "Unable to update endpoint configuration.",
+                "Error while updating endpoint configuration: %s.");
+
+        private final String code;
+        private final String message;
+        private final String description;
+
+        ErrorMessage(String code, String message, String description) {
+
+            this.code = code;
+            this.message = message;
+            this.description = description;
+        }
+
+        public String getCode() {
+
+            return ERROR_PREFIX + code;
```

Comment:
if the error prefix is appended to the error code here, why is a error prefix added ad https://github.com/wso2/identity-api-server/pull/290/files#diff-e5f569524a34d15c1fc592a34d6534d93a67d50b2ec5d0480941be74be3b96e8R51-R55?

Reply:
resolved